lib/elixir/lib/code/identifier.ex

```diff
@@ -138,6 +138,9 @@ defmodule Code.Identifier do
     Atom.to_string(atom)
   end
 
+  # TODO: Remove this clause in v1.16 when we no longer quote operator :..//
+  def inspect_as_atom(:"..//"), do: ":\"..//\""
+
```

Comment:
I don't think we should add this clause. Inspecting is fine, the issue is only on formatted code.

```suggestion
```